src/screens/Followings/FollowersScreen.js

```diff
@@ -114,8 +120,12 @@ function FollowersScreen() {
 
       <Followings
         isLoading={isLoading}
-        dataFollower={followedUsers}
-        dataUnfollowed={unfollowedUsers}
+        dataFollower={
+          discovery?.followedUsers?.length > 0 ? discovery?.followedUsers : followedUsers
+        }
+        dataUnfollowed={
+          discovery?.unfollowedUsers?.length > 0 ? discovery?.unfollowedUsers : unfollowedUsers
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Here you're passing data to the `Followings` component based on whether there are any followed or unfollowed users in the discovery context. This is a good approach as it allows the component to display the most up-to-date data. However, consider adding null checks for `discovery`, `followedUsers`, and `unfollowedUsers` to prevent potential null reference errors.
```diff
- dataFollower={
-   discovery?.followedUsers?.length > 0 ? discovery?.followedUsers : followedUsers
- }
- dataUnfollowed={
-   discovery?.unfollowedUsers?.length > 0 ? discovery?.unfollowedUsers : unfollowedUsers
- }
+ dataFollower={
+   discovery && discovery.followedUsers && discovery.followedUsers.length > 0 ? discovery.followedUsers : followedUsers
+ }
+ dataUnfollowed={
+   discovery && discovery.unfollowedUsers && discovery.unfollowedUsers.length > 0 ? discovery.unfollowedUsers : unfollowedUsers
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->